### PR TITLE
update ECK version to 1.7

### PR DESCRIPTION
**Pull request by Avilir**
This upgrade introduce k8s CRD api version v1, (which replace v1beta1) to support OCP 4.9 with k8s 1.22

Signed-off-by: Avi Layani <alayani@redhat.com>

### CODE REVIEW

**Review by ramkiperiy — DISMISSED** [review]

File: ocs_ci/templates/app-pods/eck1.7/crds.yaml [hunk 1]
```diff
@@ -0,0 +1,4466 @@
+# Source: eck-operator-crds/templates/all-crds.yaml
+apiVersion: apiextensions.k8s.io/v1
```
Comment (vasukulkarni) [inline ocs_ci/templates/app-pods/eck1.7/crds.yaml:2]: can we comment where this file is coming from either in yaml or in python file?

**Review by tiffanyn108 — DISMISSED** [review]
LGTM.

File: ocs_ci/templates/app-pods/eck1.7/crds.yaml (hunk 1 above)
Comment (vasukulkarni) [reply ocs_ci/templates/app-pods/eck1.7/crds.yaml:2]: As discussed lets move this file outside ocs-ci,:https://github.com/red-hat-storage/ocs-workloads

**Comment by Avilir** [comment]
The PR validation job and merging of this PR is hold by https://github.com/red-hat-storage/ocs-workloads/pull/7

**Review by keesturam — APPROVED** [review]

**Comment by keesturam** [comment]
LGTM

**Review by vasukulkarni — APPROVED** [review]

**Pull request merged**